pkg/base1/cockpit.js

```diff
@@ -1853,6 +1854,29 @@ function full_scope(cockpit, $, po) {
             return new DBusProxies(self, cache, String(iface), String(path_namespace));
         };
 
+        self.poll = function poll(path, iface) {
```

Comment:
Maybe the bridge can be instructed to do the polling so that there isn't any traffic unless there is a actual change.  It already 'fixes up' properties that are merely invalidated (instead of being updated), it could also optionally fix up properties that never receive notifications.